crates/hir/src/lib.rs

```diff
@@ -1414,7 +1418,7 @@ impl Struct {
             .variant_data
             .fields()
             .iter()
-            .map(|(id, _)| Field { parent: self.into(), id })
+            .map(|(id, d)| Field { parent: self.into(), id, has_default: d.has_default })
```

Comment:
That is here we can fetch it via `variant_data[id]` since we already have ` db.struct_data(self.id).variant_data` anyways